src/backend/views/Highlighter/index.js

```diff
@@ -40,7 +46,14 @@ export default function setupHighlighter(
 
   function stopInspectingNative() {
     hideOverlay();
+    removeListenersOnWindow(window);
+    iframesListeningTo.forEach(function(frame) {
+      removeListenersOnWindow(frame.contentWindow);
```

Comment:
I might want to wrap this in a try/catch as well in case the iframe becomes cross-origin after the listeners are installed.